packages/datadog_inappwebview_tracking/README.md

```diff
@@ -0,0 +1,73 @@
+## Overview
+
+This package is an extension to the [`datadog_flutter_plugin`][1]. It allows Real User Monitoring to monitor web views created by the [`flutter_inappwebview`][2] package and eliminate blind spots in your hybrid Flutter applications.
+
+> [!WARNING]
+> This plugin does not currently support Flutter Web
+
+## Instrumenting your web views
+
+The RUM Flutter SDK provides APIs for you to control web view tracking when using the [`flutter_inappwebview`][2] package.
+
+Add both the `datadog_inappwebview_tracking` package and the `flutter_inappwebview` package to your `pubspec.yaml`:
+
+```yaml
+dependencies:
+  flutter_inappwebview: ^6.1.5
+  datadog_flutter_plugin: ^2.8.0
+  datadog_inappwebview_tracking: ^1.0.0
+```
+
+### InAppWebView
+
+To instrument an `InAppWebView`, add the `DatadogInAppWebViewUserScript` to your `initialUserScripts`, and call the `trackDatadogEvents` extension method during the `onWebViewCreated` callback:
+
+```dart
+InAppWebView(
+  // Other settings...
+  initialUserScripts: UnmodifiableListView([
+    DatadogInAppWebViewUserScript(
+      datadog: DatadogSdk.instance,
+      allowedHosts: {'shopist.io'},
+    ),
+  ]),
+  onWebViewCreated: (controller) async {
+    controller.trackDatadogEvents(DatadogSdk.instance);
+  },
+)
+```
+
+### InAppBrowser
+
+> [!WARNING]
+> `InAppBrowser` is not tracked on Android 33+ because of [this bug](https://github.com/pichillilorenzo/flutter_inappwebview/issues/1973). This will be fixed by plugin versions that depends on `flutter_inappwebview 6.2.0`.
```

Comment:
If this is about future releases, I suggest avoiding promises about the future unless absolutely necessary. If it's about current versions, please change to:

> This is fixed in plugin versions...

Reply:
This is "already fixed" and planned as a release of 1.1, but because `flutter_inappwebview` is only released in beta, we'll have to do the same until it has an official release.